projects/js-packages/publicize-components/src/components/connection-management/disconnect.tsx

```diff
@@ -56,16 +63,35 @@ export function Disconnect( {
 	}
 
 	return (
-		<Button
-			size="small"
-			onClick={ onClickDisconnect }
-			disabled={ isDisconnecting }
-			variant={ variant }
-			isDestructive={ isDestructive }
-		>
-			{ isDisconnecting
-				? __( 'Disconnecting…', 'jetpack' )
-				: label || _x( 'Disconnect', 'Disconnect a social media account', 'jetpack' ) }
-		</Button>
+		<>
+			<ConfirmDialog
+				className={ styles.confirmDialog }
+				isOpen={ isConfirmOpen }
+				onConfirm={ onClickDisconnect }
+				onCancel={ toggleConfirm }
+				cancelButtonText={ __( 'Cancel', 'jetpack' ) }
+				confirmButtonText={ __( 'Yes', 'jetpack' ) }
+			>
+				{ createInterpolateElement(
+					sprintf(
+						// translators: %s: Name of the connections the user disconnects.
```

Comment:
Should this be something like:

```suggestion
						// translators: %s: The name of the connection the user is disconnecting.
```